src/components/Footer/index.tsx

```diff
@@ -26,23 +29,30 @@ function Component() {
             <p>
                 Copyright &copy; {copyrightYear} {SITE_NAME}
             </p>
-            <div className="d-flex">
+            <div className="d-flex align-items-center">
+                <TwitterShareButton
+                    className="text-primary fs-5"
+                    title="Save yourself hundreds of thousands of dollars and years wasted on a traditional MBA, and instead learn the principles that billionaires have mastered. Check out the secrets of their success at"
+                    url="https://www.contrarian.mba/"
```

Comment:
- In a future update, I  would like to be able to share to all 3 platforms. One share button that only does Twitter doesn't make as much sense.
- The URL should be inside `constants/general` and called `export const SITE_URL = 'https://www.contrarian.mba';`
- I would also like to have a Facebook like button.
- I would like to have the share/like button also show up on the Hero image, near the bottom right of it.

Reply:
- Share buttons added, but Instagram is not supporeted yet, suggest Linkedin
- done
- Will add on Next PR - The page should be an app because `react-facebook` requires appId.
- done